app/controllers/customers_controller.rb

```diff
@@ -10,6 +10,7 @@ def index
   # GET /customers/1
   # GET /customers/1.json
   def show
+        @orders = @customer.orders
```

Comment:
@SyakiraAzmi Why did this line was removed?

Reply:
It's added, not removed.

Reply:
Whoops, my bad. Hehe